### ISSUE TITLE
Accumlated issues

### ISSUE THREAD

**Issue by joetimmy** [AdguardTeam/AdGuardVPNForiOS#290]
### Current Usage:
- Using the latest AdGuard VPN version.
- Unable to find solutions in the documentation.
- No prior user reports of the same issue.
- Consistently reproducing the problem across different devices and networks.

#### Performance Problems:
- Severe performance issues with ping response times and server speed.
- For example, an 11ms ping server delivers over 100ms in speed tests.
- **Solution:** Invest in server infrastructure upgrades for improved performance.

#### Streaming Issues:
- AdGuard VPN significantly lags behind Google One VPN in performance.
- Slow operation across all devices and networks.
- Streaming problems with non-USA servers and DNS changes.
- Urgent need for universal streaming improvements.
- DNS changes disrupt streaming; likely a VPN software issue.
- **Solutions:** 
  - Optimize server performance.
  - Enhance VPN design for speed.
  - Ensure all servers unblock streaming content.
  - Address DNS-related streaming issues, expand AdGuard Paid DNS for VPN compatibility.

#### iOS Compatibility:
- iOS public networks block integrated mode, impacting compatibility.
- **Solution:** Prioritize iOS compatibility for public networks.

#### Website Blocking:
- Websites like ChatGPT flag AdGuard VPN on non-streaming servers.
- Streaming-friendly servers partially resolve the issue.
- Extensive IP bans on most servers, affecting cloudflare-protected sites.
- Most IPs are blacklisted with a 99/100 fraud score often abused by bots, cleaning up the IP also means stopping abusers of your service. 

Please see the below images for reference: 

<a href="https://ibb.co/yyjXYXT"><img src="https://i.ibb.co/yyjXYXT/Screenshot-2023-09-14-at-12-12-00-AM.png" alt="Screenshot-2023-09-14-at-12-12-00-AM" border="0"></a> <a href="https://ibb.co/Ct2TBFM"><img src="https://i.ibb.co/Ct2TBFM/Screenshot-2023-09-14-at-12-12-26-AM.png" alt="Screenshot-2023-09-14-at-12-12-26-AM" border="0"></a> <a href="https://ibb.co/0nN4CFy"><img src="https://i.ibb.co/0nN4CFy/Screenshot-2023-09-14-at-12-12-57-AM.png" alt="Screenshot-2023-09-14-at-12-12-57-AM" border="0"></a> <a href="https://ibb.co/Fwkd5r8"><img src="https://i.ibb.co/Fwkd5r8/Screenshot-2023-09-14-at-12-13-36-AM.png" alt="Screenshot-2023-09-14-at-12-13-36-AM" border="0"></a>

- **Solution:** Swiftly mitigate website blocking and IP blacklisting issues; put in anti abuse measures in place to ban bots and abusive users destroying your VPN IP reputation scores. 

### Expected Behavior:
- Seamless streaming on all servers, not limited to the USA.
- Substantial VPN speed improvements with significant ping reductions.
- Guarantee compatibility with iOS on public networks.
- Promptly address website blocking and IP blacklisting issues.

### Conclusion:
In conclusion, addressing these critical issues with AdGuard VPN is essential for providing users with a seamless and reliable online experience. While the VPN offers promising features, these performance and functionality concerns must be overcome.

### Recommendation:
I strongly urge the AdGuard VPN development team to prioritize and expedite solutions for the identified issues. Focusing on server performance enhancements, universal streaming access, iOS compatibility improvements, and rapid resolution of website blocking and IP blacklisting concerns should be paramount. By addressing these issues promptly, AdGuard VPN can establish itself as a competitive and user-friendly choice in the VPN market, delivering a genuinely seamless and secure online experience for all users.

**Comment by ESurina** [comment]
Thank you for your feedback! We will pass these comments on to the developers so they can think about what they can do about it. We are working on the application and regularly release new versions with improvements. Our developers are currently refactoring the application to improve its performance. We are also increasing the number of our servers that can support streaming. Regarding the integration mode in public networks: this is due to the fact that some networks restrict the protocol we use, so this problem is not on our side.

**Comment by joetimmy** [comment]
Hello, 

Thank you for the support, in regards to the public networks - can you automated the applications to better provide notices on this - such as your network is blocking intergrated mode or something along the lines - a prompt. 


Additionally, can you address the IP reputation issues due to abusive users? 
The IP reputation issues are a large problem with hard blacklists preventing using it seamlessly, alot of web pages blocking when using the VPN  - some very important tools. 

How is the team looking at addressing this or planning on looking into this? 

Thanks so much.
